app/models/spree/order.rb

```diff
@@ -141,6 +141,11 @@ def states
         joins('LEFT OUTER JOIN spree_products ON (spree_products.id = spree_variants.product_id)')
     }
 
+    scope :placed_by_customer_email, lambda { |email|
+      joins('LEFT OUTER JOIN customers ON (customers.id = spree_orders.customer_id)'). 
```

Comment:
I decided to join on `customers.email` and not on `order.email`, but maybe I'm wrong.
@mkllnk any tips?